tensordict/utils.py

```diff
@@ -655,7 +651,9 @@ def _get_item(tensor: torch.Tensor, index):
         return tensor[index]
 
 
-def _set_item(tensor: torch.Tensor, value, index):
+def _set_item(
+    tensor: torch.Tensor, value: torch.Tensor, index: IndexType
+) -> torch.Tensor:
```

Comment:
The function signature is kind of wared here.

```python
setitem(tensor, value, index)
setitem(tensor, index, value)
```

Reply:
It's a private method, we can change it freely if we want :)